app/client/src/components/shared/AddSaveDataWidget.SearchPanel.tsx

```diff
@@ -1,6 +1,6 @@
 /** @jsxImportSource @emotion/react */
 
-import {
+import React, {
```

Comment:
No issue here, but I tested this out and it looks like it doesn't need to be imported if it's only being used as a namespace for types.